scripts/sonic_sku_create.py

```diff
@@ -0,0 +1,853 @@
+		self.platform_specific()	
+	
+	
+
+		
+	def create_port_config(self) :
+		#create a port_config.ini file based on the sku definition 
+		if not os.path.exists(self.new_sku_dir):
+			print("Error - path:", self.new_sku_dir, " doesn't exist",file=sys.stderr)
+			exit(1)
+			
+		try:
+			f = open(self.new_sku_dir+"port_config.ini","w+")
+		except IOError:
+			print ("Could not open file "+ self.new_sku_dir+"port_config.ini", file=sys.stderr)
+			exit(1)
+		header = PORTCONFIG_HEADER # ["name", "lanes", "alias", "index"]
+		port_config = []
+		for line in self.portconfig_dict.values():
+			port_config.append(line)
+		 
+		port_config.sort(key=lambda x: (int(re.search(('\d+'),x[0]).group(0)))) # sort the list with interface name
+		f.write(tabulate(port_config, header,tablefmt="plain"))
+		f.close()
+	
+	def print_port_config(self) :
+		#print a port_config.ini file based on the sku definition 
+		
+		header = PORTCONFIG_HEADER # ["name", "lanes", "alias", "index"]
+		port_config = []
+		for line in self.portconfig_dict.values():
+			port_config.append(line)
+		 
+		port_config.sort(key=lambda x: (int(re.search(('\d+'),x[0]).group(0)))) # sort the list with interface name
+		print(tabulate(port_config, header,tablefmt="plain"))
+		
+		
+	def create_sku_dir(self) :
+		# create a new SKU directory based on the base SKU
+		if (os.path.exists(self.new_sku_dir)):
+			print ("SKU directory: "+self.new_sku_dir+ " already exists\n Please use -r flag to remove the SKU dir first", file=sys.stderr)
+			exit(1)
+		try:
+			shutil.copytree(self.base_sku_dir, self.new_sku_dir)
+		except OSError as e:
+			print (e.message, file=sys.stderr)
+	
+	def remove_sku_dir(self) :
+		# remove SKU directory 
+		DEFAULT_BASE_PATH = self.default_sku_path + '/' + "ACS-MSN2700/"
+		if (self.new_sku_dir in [self.base_sku_dir,DEFAULT_BASE_PATH]) :
+			print ("Removing the base SKU" + self.new_sku_dir + " is not allowed", file=sys.stderr)
+			exit(1)
+		try:
+			if not os.path.exists(self.new_sku_dir):
+				print ("Trying to remove a SKU "+ self.new_sku_dir + " that doesn't exists, Ignoring -r command")
+			while True:
+				answer = raw_input("You are about to permanently delete the SKU "+ self.new_sku_dir+" !! \nDo you want to continue (Yes/No)?") 
+				if (answer == "Yes" or answer == "No"):
+					break
+				else:
+					print("Valid answers are Yes or No")
+			if (answer == "Yes") :
+				shutil.rmtree(self.new_sku_dir)
+				print ("SKU directory: "+ self.new_sku_dir + " was removed")
+			else :
+				print ("SKU directory: "+ self.new_sku_dir + " was NOT removed")
+		except OSError as e:
+			print(e.message, file=sys.stderr) 
+					
+	def platform_specific(self) :
+                print("MADHAN platform ", self.platform)
+		func = self.platform_specific_dict.get(self.platform, lambda: "nothing")
+		return func()
+		
+		
+	def msn2700_specific(self) :
+                print ("MADHAN inside MSN2700 specific ")
```

Comment:
@madhanmellanox please remove debug code

Reply:
Addressed in next commit.